library/psa_crypto.c

```diff
@@ -5194,6 +5194,12 @@ psa_status_t psa_aead_update_ad(psa_aead_operation_t *operation,
         goto exit;
     }
 
```

Comment:
Please add a changelog entry. It's a minor bug, but still a bug.

Though historically we don't seem to have announced these fixes in changelog entries? I know I've fixed some before, but I can't find “null pointer arithmetic” or variations in the changelog (this is not a “null pointer dereference”).

Reply:
Its very close, but no, this is just undefined behaviour.

Reply:
Ah, yes, we've used phrasing like “Fix undefined behavior in mbedtls_ssl_read() and mbedtls_ssl_write() if len argument is 0 and buffer is NULL.”.